web/src/lib/store/auth/index.js

```diff
@@ -61,10 +61,11 @@ export const loginAction = createAsyncThunk('auth/loginAction', async ({ params,
     throw new Error(err)
   }
 
-  const { access_token, expires_in } = res
+  const { access_token, expires_in } = res // `expires_in ` is in seconds, default is 499
 
   localStorage.setItem('accessToken', access_token)
   localStorage.setItem('expiredIn', expires_in)
+  localStorage.setItem('isIdle', false)
```

Comment:
only after login idle is false, any other event action?

Reply:
[web/src/lib/provider/session.js](https://github.com/datastrato/gravitino/pull/3512/files#diff-6b423168fab9ac6f071121b402c464795d674cfd97d2aab29da8794c3becf093)#L42-L46
After exceeding the set time period, isIdle will be set to true.